src/drivers/device/integrator.h

```diff
@@ -69,7 +69,7 @@ class Integrator
 	 * @param integral_dt	Get the dt in us of the current integration (only if reset).
 	 * @return		the integral since the last read-reset
 	 */
-	math::Vector<3>		get(bool reset, uint64_t &integral_dt);
+	math::Vector<3>		get(bool reset, uint64_t *integral_dt);
```

Comment:
May I ask why this change? Any particular reason?
 
It broke `make eagle_default`.